### PR TITLE
PLANNER-848: Workbench: Provide tooltips for example import select options

### PR DESCRIPTION
**Pull request by mcimbora**
@manstis Can you please review the PR? Thanks in advance.

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/common/MenuResourceHandlerWidget.java [hunk 1]
```diff
@@ -31,9 +31,19 @@
...
-    public void init(final String title,
+    public void init(final String text,
+                     final String tooltip,
                      final Command onClick) {
-        link.setTextContent(title);
+        if (tooltip != null) {
+            link.setTitle(tooltip);
+        }
+
+        this.init(text, onClick);
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/common/MenuResourceHandlerWidget.java]: Formatting.

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/ImportProjectButtonView.java [hunk 2]
```diff
@@ -67,7 +67,15 @@ public void clearDropdown() {
     @Override
     public void addOption(final String description,
                           final Command command) {
+        addOption(description, null, command);
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/ImportProjectButtonView.java]: Formatting.

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/ImportProjectButtonView.java [hunk 3]
```diff
@@ -67,7 +67,15 @@ public void clearDropdown() {
...
+    }
+
+    @Override
+    public void addOption(String description,
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/widgets/library/ImportProjectButtonView.java]: It's nice to be consistent with the use of ```final``` modifier on function declarations... See use of ```final``` elsewhere in this PR.

**Comment by mcimbora** [comment]
@manstis PR updated. I understand this is what we all need to do.

**Comment by manstis** [comment]
Jenkins please test this (need I ask!?!)

**Comment by manstis** [comment]
Thanks @mcimbora 

Could you please let me know when it goes green...

**Comment by mcimbora** [comment]
Jenkins retest this.

**Comment by manstis** [comment]
Jenkins retest this.

**Comment by mcimbora** [comment]
Jenkins retest this.

**Comment by ge0ffrey** [comment]
For this build, this is failing:
  org.kie.workbench.common.stunner.bpmn.backend.service.indexing.BpmnFileIndexerTest.testBpmnIndexing

**Comment by manstis** [comment]
Jenkins retest this.

**Comment by mcimbora** [comment]
Latest failure org.kie.workbench.common.forms.editor.backend.indexing.FormDefinitionIndexerTest.testFormsIndexing

**Comment by mcimbora** [comment]
Jenkins retest this.

**Comment by mcimbora** [comment]
maven-bundle-plugin failed in UF:uberfire-maven-support module. 

```
19:01:27 [ERROR] Failed to execute goal org.apache.felix:maven-bundle-plugin:3.2.0:bundle (default-bundle) on project uberfire-maven-support: Execution default-bundle of goal org.apache.felix:maven-bundle-plugin:3.2.0:bundle failed. NullPointerException -> [Help 1]
19:01:27 org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.felix:maven-bundle-plugin:3.2.0:bundle (default-bundle) on project uberfire-maven-support: Execution default-bundle of goal org.apache.felix:maven-bundle-plugin:3.2.0:bundle failed.
19:01:27 	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
19:01:27 	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
19:01:27 	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
19:01:27 	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
19:01:27 	at org.apache.maven.lifecycle.internal.builder.multithreaded.MultiThreadedBuilder$1.call(MultiThreadedBuilder.java:185)
19:01:27 	at org.apache.maven.lifecycle.internal.builder.multithreaded.MultiThreadedBuilder$1.call(MultiThreadedBuilder.java:181)
19:01:27 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
19:01:27 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
19:01:27 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
19:01:27 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
19:01:27 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
19:01:27 	at java.lang.Thread.run(Thread.java:748)
19:01:27 Caused by: org.apache.maven.plugin.PluginExecutionException: Execution default-bundle of goal org.apache.felix:maven-bundle-plugin:3.2.0:bundle failed.
19:01:27 	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:145)
19:01:27 	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
19:01:27 	... 11 more
19:01:27 Caused by: java.lang.NullPointerException
19:01:27 	at org.apache.maven.shared.dependency.graph.internal.DefaultDependencyGraphBuilder.buildDependencyGraph(DefaultDependencyGraphBuilder.java:60)
19:01:27 	at org.apache.felix.bundleplugin.BundlePlugin.buildDependencyGraph(BundlePlugin.java:353)
19:01:27 	at org.apache.felix.bundleplugin.BundlePlugin.execute(BundlePlugin.java:378)
19:01:27 	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
19:01:27 	... 12 more
```

**Comment by mcimbora** [comment]
Jenkins retest this.

**Comment by mcimbora** [comment]
`6:18:02 Unexpected error while executing the Upstream repositories builder for PR builds! Failed to get open PRs for kiegroup/jbpm` with socket timeout exception. Sounds like a temporary GH issue. Let's give it another shot.

**Comment by mcimbora** [comment]
Jenkins retest this.

**Pull request merged**